tldw_Server_API/app/api/v1/endpoints/config_info.py

```
@@ -406,3 +415,333 @@ async def get_quickstart_redirect():
+        if val:
+            return val
+
+    # 2) Check get_api_keys() which reads both env and config.txt
+    try:
+        from tldw_Server_API.app.api.v1.schemas.chat_request_schemas import get_api_keys
+        keys = get_api_keys()
+        val = (keys.get(provider) or "").strip()
+        if val:
+            return val
+    except Exception:
+        pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)